src/@types/apiRequest.d.ts

```diff
@@ -0,0 +1,46 @@
+interface LoginRequestBody {
+  email: string;
+  password: string;
+}
+
+interface RegisterRequestBody extends LoginRequestBody {
+  username: string;
+}
+
+interface AddNewOrUpdateCollectionRequestBody {
+  name: string;
+  description?: string;
+  is_private?: boolean;
+}
+
+interface AddGameToCollectionRequestBody {
+  games: number[];
+}
+
+interface AddGameToFavoriteRequestBody {
+  game: number;
+  position: number;
+}
+
+interface AddToLibraryRequestBody {
```

Comment:
not required to write RequestBody as suffix in every function as you have already mention in file name